pkg/cloud/buckets/buckets.go

```diff
@@ -103,6 +103,28 @@ func ReadBucketURL(u *url.URL, timeout time.Duration) ([]byte, error) {
 	return data, nil
 }
 
+// WriteBucketURL writes the data to a bucket URL of the for 's3://bucketName/foo/bar/whatnot.txt?param=123'
+// with the given timeout
+func WriteBucketURL(u *url.URL, data []byte, timeout time.Duration) error {
```

Comment:
Would `WriteToBucketURL` be a better name? For me this makes more sense given the semantics of a bucket.